code/modules/clothing/suits/reactive_armour.dm

```diff
@@ -139,7 +140,7 @@
 		E.Copy_Parent(owner, 50)
 		E.GiveTarget(owner) //so it starts running right away
 		E.Goto(owner, E.move_to_delay, E.minimum_distance)
-		owner.alpha = 0
+		owner.alpha = 75
```

Comment:
An alpha of ~30 would make the RSA actually viable while also solving the problems that come with absolute invisibility

Reply:
kk